src/main/scala/com/itechart/ny_accidents/utils/PostgisUtils.scala

```diff
@@ -1,16 +1,18 @@
 package com.itechart.ny_accidents.utils
 
 import com.vividsolutions.jts.geom
-import com.vividsolutions.jts.geom.{Coordinate, Geometry, Point}
+import com.vividsolutions.jts.geom.{Coordinate, Geometry, Point, PrecisionModel}
 import com.vividsolutions.jts.io.WKTReader
 
 object PostgisUtils {
+  private lazy val SRID = 2163
```

Comment:
Even if that's moved to constant, it's still a magic number, because constant name does not provide a context.
Which SRID is 2163? Is it BELARUS_SRID? Is it SIBERIA_SRID?
Looking at this constant I can't say why 2163 was picked

Reply:
Renamed